.jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/c7/71564f874a8bb83404d37c922fd6727a44d1cbcbee928c1d9c82a3c2d52fb6

```diff
@@ -0,0 +1,32 @@
+I"<table>
+  <thead>
+    <tr>
+      <th>Test Instruction</th>
+      <th>Instruction Detail</th>
+      <th>Expected Test Case Result</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>IC-1</strong></td>
+      <td>Identify any <code class="language-plaintext highlighter-rouge">&lt;img&gt;</code> elements.</td>
+      <td>One <code class="language-plaintext highlighter-rouge">&lt;img&gt;</code> exists in the test data code sample.</td>
+    </tr>
+    <tr>
+      <td><strong>IC-1</strong></td>
+      <td>Determine whether the image is meaningful. Review the page content and image context to determine whether the image conveys meaningful information.</td>
+      <td>The image is meaningful, based on the text directly below the image.</td>
+    </tr>
+    <tr>
+      <td><strong>6.1</strong></td>
+      <td>Calculate the accessible name and accessible description output for the image according to <a href="https://www.w3.org/TR/html-aam-1.0/#img-element">HTML Accessibility API Mappings 1.0 for img</a>.</td>
+      <td>The output for the accessible name and accessible description computations is <code class="language-plaintext highlighter-rouge">empty</code>.</td>
+    </tr>
+    <tr>
+      <td><strong>Result</strong></td>
+      <td>Determine whether any checks fail according to the <code class="language-plaintext highlighter-rouge">6. Images</code> test instructions.</td>
```

Comment:
6.1's Baseline Result is "If any of the above checks fail, then Baseline Test 6.1-MeaningfulImage fails."

Reply:
fixed